_posts/2021-09-03-rudiments-of-data-wrangling-in-rust.md

```diff
@@ -0,0 +1,514 @@
+---
+layout: post
+title: Rudiments of Data Wrangling in Rust
+author: jerekapyaho
+excerpt: The Rust programming language is typically positioned 
+for systems programming, but there is nothing stopping us from
+enjoying it for basic data wrangling tasks like those often done in Python.
+---
```

Comment:
Would always be great to have pictures in the blogs, any pictures grab attention, even screenshots, graphs.

Reply:
I messed up the image links with wrong file extensions, sorry. They are fixed in the next commit. Also added some tags.